azext_iot/tests/deviceupdate/test_adu_account_int.py

```diff
@@ -71,11 +68,14 @@ def test_account_create_identity_system_assign_scope(provisioned_accounts: Dict[
 def test_account_create_custom():
     # Test create --no-wait/wait combo.
     target_account_name = generate_account_id()
+    # @digimaun - We can uncomment once we have more capacity in none eastus2euap location.
```

Comment:
none eastus2euap location?

Reply:
non-eastus2euap location

Reply:
plz fix so if I look at it again in a month I wouldn't be confused